### ISSUE TITLE
NVDA not reading visually hidden text on mouseover

### ISSUE THREAD

**Issue by Fenwick17** [nhsuk/nhsuk-frontend#760]
# Bug Report

## What is the issue?

When mousing over regular content with NVDA enabled, the content will be read out. 
However, when mousing over a link or content that includes visually hidden text with `nhsuk-u-visually-hidden` NVDA does not read this content out.
As an example our summary list component:
```
<a href="#">
  Change<span class="nhsuk-u-visually-hidden"> date of birth</span>
</a>
```
When this link is hovered NVDA will just read "Change" and never detect the visually hidden content within the link.

## What steps are required to reproduce the issue?

Enable NVDA on a Chromium based browser and hover content that uses `nhsuk-u-visually-hidden`

## What was the environment where this issue occurred?
Microsoft Edge Version 92.0.902.5
NVDA 2021.1
windows 10 Version 10.0.19043 Build 19043

* Device:
* Operating System: Windows 10 Version 10.0.19043 Build 19043
* Browser: Microsoft Edge Version 92.0.902.5 also latest Chrome and any other Chromium browsers
* NHS.UK frontend package version: Version 5
